tests/integration-tests/src/commonTest/kotlin/test/OtherCacheTest.kt

```diff
@@ -182,4 +183,21 @@ class OtherCacheTest {
 
     assertEquals(instant, response.data!!.instant)
   }
+
+  @Test
+  fun mutationWithObjectValueArgument() = runTest(before = { setUp() }, after = { tearDown() }) {
```

Comment:
Even more nitpicking (sorry 😅 ). It doesn't have to be a mutation but it has to be using the cache so after some more thoughts, I'd go with:

```suggestion
  fun cacheFieldWithObjectValueArgument() = runTest(before = { setUp() }, after = { tearDown() }) {
```